R/fortify_stats_lm.R

```diff
@@ -83,7 +83,9 @@ autoplot.lm <- function(object, which = c(1:3, 5), data = NULL,
     is_const_lev <- all(hii == 0) ||
       diff(hii) < 1e-10 * mean(hii, na.rm = TRUE)
 
-    fs <- dplyr::select_if(plot.data, is.factor)
+    fs <- dplyr::select_if(plot.data,
+                    function(x) is.character(x) | is.factor(x))
+    fs <- dplyr::select(fs, -.label)
```

Comment:
Just want to double check - is this line `dplyr::select(fs, -.label)` intentional? It would be good to check whether the plots look the same before and after this change for non-character variables.

Reply:
Yes, because it would select the row name (= .label) which we don't want to use as factor. The plots look the same. However I'm changing this because I got a note in checking the package.

Reply:
Ok, I will wait for your change before merge.